### DISCUSSION TITLE
x86_64 for hassio!

### DISCUSSION THREAD

**Discussion by mrcrunchybeans** [rdz-oss/BattyBirdNET-Pi#38] [Ideas]
Hey! I was wondering if I could help / learn how to make a version of this in the x86_64 format, so that it can be brought to hassio for a wider audience. Anyone have any ideas?

**Comment by rdz-oss** [comment]
Hi @mrcrunchybeans 
I do not know hassio. Just briefly scanned their info. x86_64 should work. Maybe even out of the box as the tensorfow dependency is now installed from a general package.

BR
Richard

**Comment by mrcrunchybeans** [comment]
Ultimately, I want it to run through docker on a laptop; which will be made into an add-on for home assistant.

If it were possible - that would be awesome. I'm using BirdNet-Pi now, and it works great, but I'd love to be able to listen for bats, as well.
On 7/30/2024 8:42:24 AM, Dr. Richard Zinck ***@***.***> wrote:
Hi @mrcrunchybeans [https://github.com/mrcrunchybeans]
I do not know hassio. Just briefly scanned their info. x86_64 should work. Maybe even out of the box as the tensorfow dependency is now installed from a general package.
BR
Richard
—
Reply to this email directly, view it on GitHub [https://github.com/rdz-oss/BattyBirdNET-Pi/discussions/38#discussioncomment-10191078], or unsubscribe [https://github.com/notifications/unsubscribe-auth/AUQWPFZSNYICKWDSWB6S76DZO6J35AVCNFSM6AAAAABLTQM36SVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMJZGEYDOOA].
You are receiving this because you were mentioned.Message ID: ***@***.***>
[ae15acc6-51e9-4023-9990-cfe74d46eb76]

**Reply by rdz-oss** [reply]
Dockerisation in form of a monolith should work. More complex if one would take apart the services/database etc.

**Reply by mrcrunchybeans** [reply]
... As it turns out. I don't know how to do that.

**Reply by alexbelgium** [reply]
I've made the homeassistant addons for birdnet-pi (aarch64 and amd64), birdnet-go (aarch64 and amd64) and battybirdnet-pi (aarch64 only). It would be great to convert battybirdnet-pi to amd64 but not sure how to address that and what is the limiting factor!

Edit : here is a docker for battybirdnet-pi working on aarch64 and amd64 : https://github.com/alexbelgium/hassio-addons/tree/master/battybirdnet-pi

Edit 2 : PR created for amd64

**Reply by rdz-oss** [reply]
I integrated the pull request by @alexbelgium .  I tested on an Pi 4 and install and run worked fine. Not tested myself on other machines. But might work now!

**Comment by rdz-oss** [comment]
FYI there is now a default password 'birdnet' in case it affects your docker builds downstream.

**Comment by mrcrunchybeans** [comment]
This is great. I've got it installed on HomeAssistant on proxmox. Now to 
get it configured. What was the secret sauce to get it to install 
correctly?

On 2025-04-01 15:02, Dr. Richard Zinck wrote:

> FYI there is now a default password 'birdnet' in case it affects your 
> docker builds downstream.
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you were mentioned.


Links:
------
[1] 
https://github.com/rdz-oss/BattyBirdNET-Pi/discussions/38#discussioncomment-12692850
[2]
